Example/DynamicLinks/App/iOS/FIRAppDelegate.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Google
```

Comment:
Looks like the example has 2017 as the copyright date - this should probably be updated to 2018.

Reply:
I copied the harness app from an existing one, so kept the copyright year.

Reply:
SGTM